chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/summarystatistics/SummaryStatisticsCreatorJob.java

```diff
@@ -0,0 +1,201 @@
+            Date startDate = getStartDate();
+            if (startDate == null) {
+                throw new RuntimeException("Could not obtain the startDate.");
+            }
+            Date endDate = new Date();
+            Integer numDaysInPeriod = Integer.valueOf(props.getProperty("summaryEmailSubject").toString());
+
+            Future<Statistics> futureEmailBodyStats = asynchronousStatisticsInitializor.getStatistics(null);
+            Statistics emailBodyStats = futureEmailBodyStats.get();
+
+            if (generateCsv) {
+                createSummaryStatisticsFile(startDate, endDate, numDaysInPeriod);
+            }
+            saveSummaryStatistics(emailBodyStats, endDate);
+
+        } catch (Exception e) {
+            LOGGER.error(e);
+        } finally {
+            context.close();
+        }
+    }
+
+    private void createSummaryStatisticsFile(final Date startDate, final Date endDate,
+            final Integer numDaysInPeriod) throws InterruptedException, ExecutionException {
+        List<Statistics> csvStats = new ArrayList<Statistics>();
+        Calendar startDateCal = Calendar.getInstance(TimeZone.getTimeZone(ZoneOffset.UTC));
+        startDateCal.setTime(startDate);
+        Calendar endDateCal = Calendar.getInstance(TimeZone.getTimeZone(ZoneOffset.UTC));
+        endDateCal.setTime(startDate);
+        endDateCal.add(Calendar.DATE, numDaysInPeriod);
+
+        while (endDate.compareTo(endDateCal.getTime()) >= 0) {
+            LOGGER.info("Getting csvRecord for start date " + startDateCal.getTime().toString() + " end date "
+                    + endDateCal.getTime().toString());
+            System.out.println("Getting csvRecord for start date " + startDateCal.getTime().toString() + " end date "
+                    + endDateCal.getTime().toString());
+            DateRange csvRange = new DateRange(startDateCal.getTime(), new Date(endDateCal.getTimeInMillis()));
+            Statistics historyStat = new Statistics();
+            historyStat.setDateRange(csvRange);
+            Future<Statistics> futureEmailCsvStats = asynchronousStatisticsInitializor.getStatistics(csvRange);
+            historyStat = futureEmailCsvStats.get();
+            csvStats.add(historyStat);
+            LOGGER.info("Finished getting csvRecord for start date "
+                    + startDateCal.getTime().toString() + " end date "
+                    + endDateCal.getTime().toString());
+            System.out.println("Finished getting csvRecord for start date "
+                    + startDateCal.getTime().toString() + " end date "
+                    + endDateCal.getTime().toString());
+            startDateCal.add(Calendar.DATE, numDaysInPeriod);
+            endDateCal.setTime(startDateCal.getTime());
+            endDateCal.add(Calendar.DATE, numDaysInPeriod);
+        }
+        LOGGER.info("Finished getting statistics");
+        System.out.println("Finished getting statistics");
```

Comment:
Could you make sure the System.out.printlns are removed? I think the LOGGER line should be sufficient but if you weren't able to see the logs and needed system outs for some reason then we should revisit the logging settings.